src/test/java/com/notably/model/viewstate/ViewStateModelTest.java

```diff
@@ -15,7 +15,23 @@ public void setUpBeforeEach() {
         viewStateModel = new ViewStateModelImpl();
     }
 
-    //============ HelpFlagModelModel =============================================================
+    //============ CommandInputModel ==============================================================
+    @Test
+    public void inputProperty_void_returnsValidProperty() {
+        assertNotNull(viewStateModel.inputProperty());
+    }
+
+    @Test
+    public void getInput_void_returnsValidString() {
+        assertNotNull(viewStateModel.getInput());
+    }
+
+    @Test
+    public void setInput_null_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> viewStateModel.setInput(null));
+    }
```

Comment:
If you want, I believe testing the setting/getting behavior is useful too. For example:

```java
@Test
public void setInputAndGetInput() {
  final String input = "This is the new input";
  viewStateMode.setInput(input);
  assertEquals(input, viewStateModel.getInput());
}
```

Reply:
Alright! Useful indeed :)